### ISSUE TITLE
"Playback" automatically selects 8 channel view

### ISSUE THREAD

**Issue by gabrielibagon** [OpenBCI/OpenBCI_GUI_v2.0#1]
I noticed that when playing back a 16 channel recording on the GUI, it automatically sets the GUI to read the first 8 channels of the data only. This could only be changed to the 16 channel version of the GUI when first selecting "Live (from OpenBCI)", selecting 16 channels, then clicking "Playback (from file)" and starting the system from there. This is unintuitive, since the option of "Channel Count" looks like it refers to the "Live" data source mode, but not to the "Playback" mode.

In short - the "Channel Count" option should also appear at the "Playback (from file)" panel, or have the program automatically detect that there are 16 channels (columns) of data being read.

**Comment by cfausn** [comment]
This was fixed over summer by me
